tests/_utils/run_services

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+set -eu
+
+PD_ADDR="127.0.0.1:2379"
+TIDB_IP="127.0.0.1"
+TIDB_PORT="4000"
+TIDB_ADDR="127.0.0.1:4000"
+TIDB_STATUS_ADDR="127.0.0.1:10080"
+
+stop_services() {
+    killall -9 tidb-server || true
+
+    find "$DUMPLING_TEST_DIR" -maxdepth 1 -not -path "$DUMPLING_TEST_DIR" -not -name "*.log" | xargs rm -r || true
+}
+
+start_services() {
+    stop_services
+    echo "Starting TiDB..."
+
+    echo "Ensure mysql is started..."
+    i=0
+    while ! run_sql 'select 0 limit 0' > /dev/null; do
+       i=$((i+1))
+       if [ "$i" -gt 10 ]; then
+          echo 'Failed to ping MySQL Server'
+          exit 1
+       fi
+       sleep 3
+    done
+
+    echo "Starting PD..."
```

Comment:
does `mocktikv tidb` need `pd`?

Reply:
It's lightning needs pd to start

Reply:
You can turn off `--check-requirements`

Reply:
okay, let me try

Reply:
when I removed pd-server, it seems lightning will try to get settings from tidb, then it will get this error:
```
[error="cannot decode settings from TiDB, please manually fill in `tidb.port` and `tidb.pd-addr`: json: cannot unmarshal bool into Go struct field Log.log.enable-slow-log of type uint32"]
```

seems tidb has changed log.EnableSlowLog from uint32 to bool in master
we need update tidb dependency in lightning first.
then use latest lightining

Reply:
Ok, please file an issue to Lightning to track it.

Reply:
Sure,have filed an [issue](https://github.com/pingcap/tidb-lightning/issues/277)

Reply:
maybe I can keep pd-server here in this PR, leave TODO to remove it in the future. @kennytm after lightning fix the issue, I'll use another PR to fix it.

Reply:
Found a proper version (v3.1.0.beta.1) that has no such problem. CI fixed PTAL